x/gamm/keeper/pool_service.go

```diff
@@ -189,12 +192,13 @@ func (k Keeper) JoinPoolNoSwap(
 		return err
 	}
 
+	// we do an abstract calculation on the lp liquidity coins needed to have the designated amount of given shares of the pool
```

Comment:
Does it make sense to elaborate by talking about the fact that no swap would be done?

Reply:
make sense. added a line here, lmk what you think or if you have better ideas for the wording!